fusor-ember-cli/app/templates/loading.hbs

```diff
@@ -1 +1,8 @@
-{{loading-spinner show=true text="Loading..."}}
```

Comment:
reverted loading.hbs template. I prefer to discuss this with UX team before changing.